### ISSUE TITLE
[Bug] Click any tweets in timeline it will crash [1.0.1]

### ISSUE THREAD

**Issue by chenshaoju** [TwidereProject/TwidereX-Android#30]
# Bug Report

## Environment

Network type:

- [ ] Cellular
- [X] Wi-Fi

## Build Variant

- [X] Google Play
- [ ] FDroid

App version: 
1.0.1

## Bug Info
Click any tweets in timeline, The Twidere X will crash

### Steps to reproduce
1.Open Twidere X
2.In timeline, Click any tweet.

### Actual Behavior

It's crashed.

###  Expected behavior

Show a tweet details.

### Crash logs
```
--------- beginning of crash
01-18 13:10:45.039  6011  6011 E AndroidRuntime: FATAL EXCEPTION: main
01-18 13:10:45.039  6011  6011 E AndroidRuntime: Process: com.twidere.twiderex, PID: 6011
01-18 13:10:45.039  6011  6011 E AndroidRuntime: MicroBlogException(error=null, request=null, errors=null, httpCode=null)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at com.twidere.services.http.MicroBlogException$$serializer.deserialize(Unknown Source:179)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at com.twidere.services.http.MicroBlogException$$serializer.deserialize(MicroBlogException.kt:27)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:63)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:51)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at kotlinx.serialization.json.internal.TreeJsonDecoderKt.readJson(TreeJsonDecoder.kt:24)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at kotlinx.serialization.json.Json.decodeFromJsonElement(Json.kt:104)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at com.twidere.services.twitter.TwitterService$resources$2$1.invoke(TwitterService.kt:408)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at com.twidere.services.twitter.TwitterService$resources$2$1.invoke(TwitterService.kt:58)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at com.twidere.services.twitter.TwitterService$resources$2$$special$$inlined$-addInterceptor$2.intercept(OkHttpClient.kt:563)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at com.twidere.services.twitter.TwitterService$resources$2$$special$$inlined$-addInterceptor$1.intercept(OkHttpClient.kt:1080)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at com.twidere.services.http.AuthorizationInterceptor.intercept(AuthorizationInterceptor.kt:36)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:201)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:517)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
01-18 13:10:45.039  6011  6011 E AndroidRuntime:        at java.lang.Thread.run(Thread.java:764)
01-18 13:14:12.464  7505  7505 E AndroidRuntime: FATAL EXCEPTION: main
01-18 13:14:12.464  7505  7505 E AndroidRuntime: Process: com.twidere.twiderex, PID: 7505
01-18 13:14:12.464  7505  7505 E AndroidRuntime: MicroBlogException(error=null, request=null, errors=null, httpCode=null)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at com.twidere.services.http.MicroBlogException$$serializer.deserialize(Unknown Source:179)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at com.twidere.services.http.MicroBlogException$$serializer.deserialize(MicroBlogException.kt:27)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic(Polymorphic.kt:63)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at kotlinx.serialization.json.internal.AbstractJsonTreeDecoder.decodeSerializableValue(TreeJsonDecoder.kt:51)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at kotlinx.serialization.json.internal.TreeJsonDecoderKt.readJson(TreeJsonDecoder.kt:24)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at kotlinx.serialization.json.Json.decodeFromJsonElement(Json.kt:104)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at com.twidere.services.twitter.TwitterService$resources$2$1.invoke(TwitterService.kt:408)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at com.twidere.services.twitter.TwitterService$resources$2$1.invoke(TwitterService.kt:58)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at com.twidere.services.twitter.TwitterService$resources$2$$special$$inlined$-addInterceptor$2.intercept(OkHttpClient.kt:563)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at com.twidere.services.twitter.TwitterService$resources$2$$special$$inlined$-addInterceptor$1.intercept(OkHttpClient.kt:1080)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at com.twidere.services.http.AuthorizationInterceptor.intercept(AuthorizationInterceptor.kt:36)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:201)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:517)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
01-18 13:14:12.464  7505  7505 E AndroidRuntime:        at java.lang.Thread.run(Thread.java:764)
01-18 13:14:12.471  7505  8125 D OSTracker: OS Event: crash
01-18 13:14:12.471  1316  3565 D Embryo_Uterus: over crash limit, add com.twidere.twiderex to black list
```

**Comment by Tlaster** [comment]
Seems like the API key for Twidere X hit the limit of monthly tweet cap usage.

**Comment by edent** [comment]
Yup. Seems to happen when getting replies.

**Comment by Tlaster** [comment]
Fixed in 1.0.2, but only the crash issue.
I'm working on a workaround that fetch the replies.
